extensions/label/json-schema/schema.json

```diff
@@ -1,119 +1,135 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema#",
-    "id": "stac-label-item-schema.json#",
-    "title": "STAC Label Data Item Extension Spec",
-    "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "stac-label-item-schema.json#",
+  "title": "STAC Label Data Item Extension Spec",
+  "type": "object",
+  "properties": {
     "properties": {
+      "type": "object",
+      "required": [
+        "label:properties",
+        "label:classes",
+        "label:description"
+      ],
       "properties": {
-        "type": "object",
-        "required": [
-          "label:properties",
-          "label:classes",
-          "label:description"
-        ],
-        "properties": {
-          "label:properties": {
-            "title": "Property",
-            "type": "array",
-            "items": {
-              "type": "string"
+        "label:properties": {
+          "title": "Property",
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "type": "null"
             }
-          },
-          "label:classes": {
-            "title": "Classes",
-            "type": "array",
-            "items": {
-              "type": "object",
-              "required": [
-                "name",
-                "classes"
-              ],
-              "properties": {
-                "name": {
-                  "title": "Name",
-                  "type": "string"
-                },
-                "classes": {
-                  "title": "Classes",
-                  "type": "array",
-                  "items": {
-                    "type": "string"
+          ]
+        },
+        "label:classes": {
+          "title": "Classes",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": [
+              "name",
```

Comment:
The doc says name and classes aren't required. Either fix it here or in the docs.

Reply:
I don't think they should be required, so this should be fixed in the docs.

Reply:
@matthewhanson given your comment, I'm removing `required` here, though I'm not certain that's the right thing given that they are currently required here and not in the docs.

Reply:
I think the required should be in place and be added to the docs. How does an empty object makes any sense? Now you can specify [{}, {]] and it's valid?!

Reply:
Required in both as of 04edb4c